### ISSUE TITLE
ZeroDivisionError occurs in gripper_v6_dxl_controller.launch

### ISSUE THREAD

**Issue by YutoUchimi** [start-jsk/jsk_apc#2182]
```
[INFO] [WallTime: 1498724090.568790] [0.000000] port: Found 4 motors - 2 AX-18 [2, 4], 2 MX-28 [1, 3], initialization complete.
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/uchimi/ros/jsk_apc/src/FOR_GRIPPER_V5/dynamixel_motor/dynamixel_driver/src/dynamixel_driver/dynamixel_serial_proxy.py", line 260, in __update_motor_states
    rates.append(1.0 / (current_time - last_time).to_sec())
ZeroDivisionError: float division by zero

[INFO] [WallTime: 1498724090.639025] [0.000000] port controller_spawner: All services are up, spawning controllers...
```

**Comment by YutoUchimi** [comment]
When I reboot Baxter, this was fixed

**Comment by pazeshun** [comment]
When this happens, we have to fix motor mode like #2180
